### PR TITLE
credhub.yml changes that allowed 'bosh create-env' to render templates

### PR DESCRIPTION
**Pull request by drnic**

### CODE REVIEW

**Comment by rkoster** [comment]
duplicate of https://github.com/cloudfoundry/bosh-deployment/pull/37

File: credhub.yml [hunk 1]
```diff
@@ -26,8 +26,13 @@
           database: credhub
         tls: ((credhub_tls))
         encryption:
-          provider: dev_internal
-          dev_key: ((credhub_dev_key))
+          keys:
+            - provider_name: dev_internal
+              active: true
+              encryption_password: ((credhub_dev_key))
```
Comment (rkoster) [inline credhub.yml]: `encryption_password` should be `dev_key`:
```yaml
dev_key: ((credhub_dev_key))
```

File: credhub.yml [hunk 2]
```diff
@@ -26,8 +26,13 @@
...
+          providers:
+            - name: dev_internal
+              dev_key: ((credhub_dev_key))
```
Comment (rkoster) [inline credhub.yml]: `dev_key` should be specified under `keys`
missing `type`:
```yaml
type: dev_internal
```

**Comment by drnic** [comment]
Closing in lieu of #37 which I hadn't seen.

**Pull request closed without merging**